core/domain/classifier_services.py

```diff
@@ -127,6 +127,75 @@ def classify_string_classifier_rule(state, normalized_answer):
     return None
 
 
+def handle_classifier_training_job_creation(exploration, state_names):
```

Comment:
Why not just call this create_classifier_training_jobs?

Actually I'm not sure why this and create_multi_classifier_training_jobs are separate functions. Is the latter ever called outside this flow? If not, why not just combine both into one function?